### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by kael89**
Changes as per the commit history

### CODE REVIEW

**Review by EMcQ-BES — APPROVED** [review]
All pretty minor, lgtm

File: packages/devops/scripts/deployment/checkoutLatest.sh [hunk 1]
```diff
@@ -3,6 +3,7 @@
 # Get latest code and dependencies
 echo "Checking out ${BRANCH}, or dev if that doesn't exist"
 cd ${HOME_DIRECTORY}
+git remote prune origin
```
Comment (kael89) [inline packages/devops/scripts/deployment/checkoutLatest.sh:6]: @edmofro this should solve the issue where deleted branches are can still be checked out during deployment

Will test that in a feature instance

**Comment by kael89** [comment]
@EMcQ-BES added a couple of small changes after you last reviewed, could you please re-review? 🙂

**Review by EMcQ-BES — APPROVED** [review]
Looks like you've just added a couple of prunes since I last looked at this, all looks sensible 👍

**Pull request merged**